docs/config.toml

```diff
@@ -179,6 +179,12 @@ url = "https://github.com/aws/eks-anywhere"
 icon = "fab fa-github"
 desc = "Development takes place here!"
 
+[[params.versions]]
+fullversion = "v0.22"
+version = "v0.22"
+docsbranch = "main"
+url = "https://anywhere.eks.amazonaws.com"
+
 [[params.versions]]
 fullversion = "v0.21"
 version = "v0.21"
```

Comment:
Do we need to change this docsbranch?

Reply:
yes I was just checking this, I think we shoud set latest branch to main